runners/spark/src/main/java/org/apache/beam/runners/spark/SparkTransformOverrides.java

```diff
@@ -23,13 +23,20 @@
 import org.apache.beam.runners.core.construction.PTransformTranslation;
 import org.apache.beam.runners.core.construction.SplittableParDo;
 import org.apache.beam.runners.core.construction.SplittableParDoNaiveBounded;
+import org.apache.beam.runners.core.construction.UnsupportedOverrideFactory;
 import org.apache.beam.sdk.runners.PTransformOverride;
 import org.apache.beam.sdk.transforms.PTransform;
 
 /** {@link PTransform} overrides for Flink runner. */
 public class SparkTransformOverrides {
   public static List<PTransformOverride> getDefaultOverrides(boolean streaming) {
     ImmutableList.Builder<PTransformOverride> builder = ImmutableList.builder();
+    // TODO: [BEAM-5358] Support @RequiresStableInput on Spark runner
```

Comment:
have beam community have plan for support `@RequiresStableInput` annotation for SparkRunner?

Reply:
I do not believe there is a plan, but feel free to ask on the dev@ email list.